<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewVendas</class>
 <widget class="QMainWindow" name="viewVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Switch Dreams</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>651</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QPushButton" name="btnVerIngressosVendidos">
         <property name="text">
          <string>Ver Ingressos Vendidos</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnComprarIngresso">
         <property name="text">
          <string>Comprar Ingresso</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btnHome">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Menu Principal de Vendas</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="btnComprar">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Comprar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Compra de Ingresso</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVoltar">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>316</width>
        <height>58</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cõdigo da Apresentação</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Quantidade</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineQuantidade"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxApr"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelCompra">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>msg de compra</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina Ingressos Vendidos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVoltar2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEve">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableQtdVendida">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>471</width>
        <height>281</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>160</number>
      </attribute>
      <column>
       <property name="text">
        <string>Apresentação</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ingressos Vendidos</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Informações</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Código de Evento</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableWidget" name="tableCpfQte">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>180</number>
      </attribute>
      <column>
       <property name="text">
        <string>Usuário</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantidade de Ingressos</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingressos Comprados</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
